.circleci/config.yml

```diff
@@ -491,6 +502,9 @@ workflows:
       - build:
           <<: *context
           <<: *not-on-integ-branch
+          matrix:
```

Comment:
You may want to prepend:
```
name: build-with-redis-<<matrix.redis_version>>
```
so you'll know what's running.

Reply:
the redis_version on the already in the name
```
ci/circleci: build-6.2
ci/circleci: build-7.0-rc2
```